test/runtime/Policies.go

```diff
@@ -668,13 +671,11 @@ var _ = Describe("RuntimePolicies", func() {
 
 		// Delete the pseudo-host IPs that we added to localhost after test
 		// finishes. Don't care about success; this is best-effort.
-		cleanup := func() {
```

Comment:
Why don't you remove this function and always run the `_ = vm.RemoveIPFromLoopbackDevice` in the `AfterEach`?

Reply:
Because that function only affect to one test, if you remove more than one time it will fail for all test. With the cleanup test will be only once.